azure/durable_functions/models/actions/ActionType.py

```diff
@@ -11,4 +11,5 @@ class ActionType(IntEnum):
     CONTINUE_AS_NEW: int = 4
     CREATE_TIMER: int = 5
     WAIT_FOR_EXTERNAL_EVENT: int = 6
+    CALL_ENTITY: int = 7
```

Comment:
We may want to save this change for a separate PR.

Reply:
Good catch, removed! I was playing with entities in Python but decided to hold on to that